prep-html-script-tag/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Script Tag</title>
+  </head>
+  <body>
+    <script>
+      console.log('I have added JS to a script tag in the HTML!');
+    </script>
+    <script src="main.js"></script>
+    <script>
+      console.log('I have added a script file via the src attribute!');
+    </script>
```

Comment:
You can delete this. We only want 2 `script` tags. The one you have on lines 7-9, and the one you have on line 10

Reply:
I fixed this, thank you!